src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -24,6 +27,9 @@ function SciMLBase.symbolic_discretize(pdesys::ModelingToolkit.PDESystem,discret
         dx = discretization.dxs[findfirst(dxs->isequal(x.val, dxs[1].val),discretization.dxs)][2]
         dx isa Number ? (xdomain.domain.lower:dx:xdomain.domain.upper) : dx
     end
+    dxs = map(nottime) do x        
```

Comment:
rebase?

Reply:
I am not sure about the meaning of the question.

Reply:
edit the PR, change the base branch from SciML:master to anything else, then edit again and change back to SciML:master. This will get rid of changes like this which are just git struggling to keep track (maybe there is a smarter way haha)

Reply:
Done :)